### PR TITLE
[unipool-xsushi] xSushi voting power strategy

### PR DESCRIPTION
**Pull request by 0xgon**
Changes proposed in this pull request:

Add a new strategy called unipool-xsushi
This strategy will give  1 voting power to token holders, 2 voting powers to LP token holders

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import strategies from './strategies';
-import utils from './utils';
+import strategies from './strategies'
+import utils from './utils'
 
 export default {
   strategies,
   utils
-};
+}
```
Comment (ChaituVR) [inline src/index.ts:7]: Why these changes

File: src/strategies/index.ts [hunk 2]
```diff
@@ -427,40 +429,40 @@ const strategies = {
...
-  strategies[strategyName].schema = schema;
-  strategies[strategyName].about = about;
-});
+  strategies[strategyName].examples = examples
+  strategies[strategyName].schema = schema
+  strategies[strategyName].about = about
+})
 
-export default strategies;
+export default strategies
```
Comment (ChaituVR) [inline src/strategies/index.ts:468]: Try to run lint command

File: src/strategies/unipool-xsushi/index.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import { multicall } from '../../utils'
+import { formatUnits } from '@ethersproject/units'
+import { BigNumber } from '@ethersproject/bignumber'
+
+export const author = 'dapplion'
```
Comment (ChaituVR) [inline src/strategies/unipool-xsushi/index.ts:5]:
```suggestion
export const author = '0xgon'
```

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,5259 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ChaituVR) [inline yarn.lock]: Revert changes from yarn.lock

File: src/strategies/unipool-xsushi/index.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    return stakedLpTokens.mul(totalTokensInPool).div(lpTokenTotalSupply) // decimal: options.decimal
+  })
+
+  // How much rewarded tokens user have in the unipool contract
+  const usersEarnedTokensList = res.slice(addresses.length).map(num => {
+    return bn(num) // decimal: options.decimal
+  })
+
+  const sumList = usersTokensFromLp.map((userTokensFromLp, i) => {
+    return userTokensFromLp.mul(2).add(usersEarnedTokensList[i])
```
Comment (ChaituVR) [inline src/strategies/unipool-xsushi/index.ts:65]: Hi @0xgon This strategy is the same as `unipool-univ2-lp` except this `mul(2)` why not use that strategy? any specific reason/usecase?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by bonustrack — APPROVED** [review]
